src/components/Tabs.vue

```diff
@@ -101,7 +103,7 @@
     data() {
       return {
         active: this.readme ? 'readme' : 'console',
-        tabHeight: 280,
+        tabHeight: defaultBottomPanelHeight,
```

Comment:
then we don't need to mutate that, just map the defaultBottomPanelHeight from store